### PR TITLE
Week 3 PR

### PR DESCRIPTION
**Pull request by nvergel**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -20,24 +20,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
```
Comment (bengaskagoogle) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: Remove this newline

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 <html>
+
   <head>
     <meta charset="UTF-8">
     <title>Nicolas Perez Vergel</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+
```
Comment (bengaskagoogle) [inline portfolio/src/main/webapp/index.html:10]: Remove space

Comment (bengaskagoogle) [reply portfolio/src/main/webapp/index.html:10]: And above

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -49,9 +51,6 @@ <h1> Personal Projects: </h1>
             <a href="http://focofoodie.com/"> FOCO House Pricing </a>
           </p>
         </div>
-        <script>
-          collapsible() //This funcion adds a listener to .collapsble which hides/unhides .content
```
Comment (bengaskagoogle) [inline portfolio/src/main/webapp/index.html:53]: Typos in 'collapsible' in comment

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,6 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*
+ * Runs two functions on load
```
Comment (bengaskagoogle) [inline portfolio/src/main/webapp/script.js:16]: Make this one line if it fits easily on one line.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (nvergel) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: done

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (nvergel) [reply portfolio/src/main/webapp/index.html:10]: done

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (nvergel) [reply portfolio/src/main/webapp/script.js:16]: done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -39,12 +40,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
     PreparedQuery results = datastore.prepare(query);
 
-    ArrayList<Person> messages = new ArrayList<Person>();
+    ArrayList<Pair<String, String>> messages = new ArrayList<Pair<String, String>>();
     for (Entity entity : results.asIterable()) {
       String name = (String) entity.getProperty("name");
       String text = (String) entity.getProperty("text");
 
-      Person message = new Person(name, text);
+      Pair<String, String> message = new Pair<String, String>(name, text);
```
Comment (bengaskagoogle) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: I personally think that the named type here, is better than the unnamed Tuple in this case. 

Person gives a lot more clear information about the purpose of the type, and also allows for easier extension later on if needed. 

What made you decide to switch to using a Pair<T, V> here?

Comment (nvergel) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: Grady said I should become familiar with Pairs for coding interviews so I felt this was good practice.

**Pull request merged**